Makefile

```diff
@@ -24,12 +22,26 @@ fioctl-%:
 		go build $(LDFLAGS) -o bin/$@ main.go
 	@if [ "$@" = "fioctl-windows-amd64" ]; then mv bin/$@ bin/$@.exe; fi
 
+GOLANGCI_LINT = $(shell pwd)/bin/golangci-lint
+golangci-lint:
+	@[ -f $(GOLANGCI_LINT) ] || { \
+	set -e ;\
+	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(shell dirname $(GOLANGCI_LINT)) v1.54.2 ;\
+	}
```

Comment:
I'm fine to have this target if it is improved a bit:

1. [required] Install a specific linter version, which is _exactly the same_ as the one used in CI.
2. [recommended] Remove the dependency between lint and `golangci-lint`.
    I call `make check` quite often, and making it run a code which _does nothing_ (except for the first time) almost useless.
    An alternative would be for this target to auto-detect a necessity to upgrade the linter.
    But then comes a question how to keep our many repos in sync.
    For a newbie it would be enough to tell smth like `Please, run "make golangci-lint" to install the linter` when they call the `make check` and there is no linter installed.
    I'd even call it `make dev-deps` in kind if we need to install something else in the future.

Reply:
> [required] Install a specific linter version, which is exactly the same as the one used in CI.
Are you speaking about the job? https://ci.foundries.io/projects/fioctl/builds/574/unit-test/
If so, can you explain why we cannot update the linter version used?

Reply:
1 - Done
2 - I am not sure if I could follow up on your suggestion.

If we call make lint or make check (which calling the lint now) it will download the lint into the bin directory so that we do not need to download it manually. If the linter exists already then we just ignore 
So, please let me know what makes you not happy then I will try to accommodate it for you.

Reply:
Your new changes were minimal enough to comment on them in a form of a suggestion.
Please, see the related comment below and let's talk in Slack if you disagree.